hotbar.coffee

```diff
@@ -92,16 +94,28 @@ class InventoryHotbarClient extends InventoryHotbarCommon
   enable: () ->
     @inventoryWindow.container.style.visibility = ''
     @onSlots = {}
+    @console.on 'open', =>
+      @scrollLock = false;
```

Comment:
Actually, I think there's a better way to do this. Try replacing your `@scrollLock` and voxel-console events with `@game.shell.pointerLock` - it seems to already do what you want here, and is more general than just having the console open. For example, it also takes into account when another modal dialog is open (e.g., inventory) - it is only true when the browser has acquired pointer lock (to interact with the game with mouse look).